### DISCUSSION TITLE
手遊的性質

### DISCUSSION THREAD

**Discussion by Terryyyyyyyyyyyy** [1101-GS4719-C/Python-Community-GS4719-C#1856] [General]
最近回鍋了一個名叫部落衝突的手遊，主要內容是升級自己的部落跟攻打別人的部落，算是養成跟塔防性質的遊戲。另外我還有玩另一款叫一念逍遙的遊戲，我認為算是放置性質的遊戲。我認為以上兩款遊戲都讓我能在閒暇時間獲取快樂。
想問大家平常都在玩什麼類型的手遊呢。

**Comment by sorn32** [comment]
最近在玩重返未來1999，我很喜歡抽卡類的遊戲，而它跟我到目前為止玩過的，有一個最大的不同，就是它是單機遊戲!!!
我覺得沒有排行榜那些，更可以好好放鬆的玩｡:.ﾟヽ(*´∀`)ﾉﾟ.:｡

**Comment by BeaGle0928** [comment]
我覺得現在的手遊好像已經不像以前這麼的有創意會很吸引玩家的目光。我曾經嘗試了下載幾款不一樣的手遊，但他們的玩法基本上都大同小異，只不過是把各種遊戲的一些部分融合成為另一個遊戲，對於遊戲環境而言，我覺得這個風氣是不健康的。

**Comment by chiuwangwang** [comment]
我是糞遊重度者，IG或YOUTUBE 推薦的廣告遊戲，我都會下載起來玩，但通常熱戀期約一個禮拜。我覺得那些遊戲都很會偷時間，一下子時間就在這些遊戲裡消失了。我只能說，我自制力不足，都會被廣告吸引。

**Comment by YANRONG0123** [comment]
我比較玩一些已經存活幾年的手遊，畢竟能活到現在一定都有各自的理由，雖然很多存在已久的手遊，玩法已經和原本的不一樣了，但人氣依舊居高不下，也不會覺得自己是孤獨的。

**Comment by woohsin** [comment]
可以暫停的小遊戲：twenty、踩地雷之類的

**Comment by 110101501** [comment]
個人蠻喜歡玩怪物彈珠。這一款遊戲時常跟日本動漫作品合作。當初是因為這款遊戲有跟咒術迴戰合作才玩，一個合作檔期結束後沒多久就又會有另一個合作。也因為如此，自己不知不覺也玩了兩年左右。推薦給喜歡動漫作品的同學試試看。

**Reply by 110101501** [reply]
順帶一提，這次的合作對象是東京喰種。蠻意外是跟老IP合作。
雖然強度不強，但因為為了收藏角色以及回味，還是會投一些寶珠抽。

**Comment by wanka2030** [comment]
手遊應該要讓人放鬆以及紓解壓力的，但大多數遊戲還是會引進排位賽、天梯或者積分來讓遊玩者變成上癮或是沈迷。但我還是會經常玩聯盟戰棋，雖然牌位一直卡住，但還是能在繁瑣繁忙的生活之中，留有著片刻的溫存。

**Comment by William08290829** [comment]
我平常都玩 MOBA（多人線上戰術遊戲）的 5v5 類型。這些遊戲需要團隊合作、策略思考和技巧操作，讓我非常著迷。但此類遊戲容易玩到心情越來越躁，尤其是隊友不給力然後又連敗的時候，所以十分不推薦想要靠玩遊戲抒發情緒或是紓解壓力的人來玩。

**Comment by EPHMR0524** [comment]
我平常有在玩的手遊都是音遊或是養成類遊戲，沒有像是Moba或fps那種要組隊的手遊，畢竟這種情況打得爛就是自己的問題，不會有被隊友氣死的狀況出現，但缺點就是對手機或平板的螢幕非常傷，我覺得我的平板已經快被砸壞了

**Comment by jylin34** [comment]
現在手機有的就只有brawl stars，這款遊戲每場時間很短，而且可以組隊，匹配時間也短，外出 或跟同學無聊時，都可以打個幾場殺時間，非常推薦。

**Comment by thomasxshen** [comment]
我有在玩爐石戰記，很好玩很有趣，也能殺時間，只是台灣玩的人有點少阿，讓我很孤單，希望大家能都來玩一下。

**Comment by YOGA012** [comment]
我都玩monster hunter now，跟pokemon go一樣都是niantic出的手遊，兩者玩法類似，都要走到戶外，最大的不同是monster hunter now是跟地圖上的魔物搏鬥，遊玩體驗上較為刺激；而pokemon go就只是單純丟球以及點擊螢幕，而且想抓到心儀的寶可夢可能抓好幾天都抓不到，原本我也有玩pokemon go，但長久下來我的熱情也被消磨殆盡，現在都不玩了。
